index.html

```diff
@@ -16,298 +16,390 @@
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hesitate to contact me.
-        </p>
-
-        <div class="about-li">
-          <img class="" src="images/Frame-1.svg" alt="side-image" />
-
-          <button class="btn ab-btn-btn" type="button">Get my Resume</button>
-        </div>
-      </div>
-      <div class="ab-lang card-flex">
-        <img class="ab-img" src="images/Icons1.svg" alt="side-image" />
-        <h2 class="ab-title new">Languages</h2>
-        <ul class="list flex-tool ab-li p-lang">
-          <li>
-            <button class="btn-btn ab-btn" type="button">Javascript</button>
-          </li>
-          <li><button class="btn-btn ab-btn" type="button">Ruby</button></li>
-          <li><button class="btn-btn ab-btn" type="button">Html</button></li>
-          <li><button class="btn-btn ab-btn" type="button">Css</button></li>
-        </ul>
-      </div>
-      <div class="ab-lang card-flex">
-        <img class="ab-img" src="images/lang2.svg" alt="side-image" />
-        <h2 class="ab-title new">Frameworks</h2>
-        <ul class="list flex-tool ab-li p-lang">
-          <li>
-            <button class="btn-btn ab-btn" type="button">Bootstrap</button>
-          </li>
-          <li>
-            <button class="btn-btn ab-btn" type="button">Ruby on rails</button>
-          </li>
-          <li><button class="btn-btn ab-btn" type="button">RSpec</button></li>
-          <li>
-            <button class="btn-btn ab-btn" type="button">Capybara</button>
-          </li>
-          <li>
-            <button class="btn-btn ab-btn" type="button">Selenium</button>
-          </li>
-        </ul>
-      </div>
-      <div class="ab-lang card-flex">
-        <img class="ab-img" src="images/lang1.svg" alt="side-image" />
-        <h2 class="ab-title new">Skills</h2>
-        <ul class="list flex-tool ab-li p-lang">
-          <li>
-            <button class="btn-btn ab-btn" type="button">codekit</button>
-          </li>
-          <li>
-            <button class="btn-btn ab-btn" type="button">Github</button>
-          </li>
-          <li>
-            <button class="btn-btn ab-btn" type="button">codepen</button>
-          </li>
-          <li>
-            <button class="btn-btn ab-btn" type="button">Gitlab</button>
-          </li>
-          <li>
-            <button class="btn-btn ab-btn" type="button">Terminal</button>
-          </li>
-        </ul>
-      </div>
-    </section>
-
     <!-- contact me section -->
     <section id="abt-contact">
-      <div class="close">
-        <h1 class="closure">
-          I'm always interested in hearing about new projects, so if you'd like
-          to chat please get in touch.
-        </h1>
-      </div>
+      <div class="contact-me">
+        <div class="contact-sec">
+          <div class="close">
+            <h1 class="closure">
```

Comment:
:red_circle: [OPTIONAL] This is a paragraph and not a heading. Kindly make sure you replace this `h1` tag with the right semantic tag which is a `p` tag